### PR TITLE
Fix: scheduled executions data

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Path, method, body and header are missing in scheduled executions.

They also are never tested properly

User ID and JWT also likely doesn't work well, and also isn't tested

## Test Plan

- [x] update & add tests

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: src/Appwrite/Platform/Tasks/ScheduleExecutions.php [hunk 1]
```diff
@@ -45,23 +45,27 @@ protected function enqueueResources(Group $pools, Database $dbForConsole): void
...
-            $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
+            $data = $dbForConsole->getDocument(
+                'schedules',
+                $schedule['$id'],
+            )->getAttribute('data', []);
 
+            $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
 
-            \go(function () use ($queueForFunctions, $schedule, $delay) {
+            \go(function () use ($queueForFunctions, $schedule, $delay, $data, $dbForConsole) {
```
Comment (ItzNotABug) [inline src/Appwrite/Platform/Tasks/ScheduleExecutions.php:55]: `$dbForConsole` isn't used in here, right?

**Review by ItzNotABug — COMMENTED** [review]
one comment, rest LGTM.

File: src/Appwrite/Platform/Tasks/ScheduleExecutions.php (hunk 1 above)
Comment (Meldiron) [reply src/Appwrite/Platform/Tasks/ScheduleExecutions.php:55]: nice catch, leftover

Comment (Meldiron) [inline src/Appwrite/Platform/Tasks/ScheduleExecutions.php:55]:
```suggestion
            \go(function () use ($queueForFunctions, $schedule, $delay, $data) {
```

**Review by ItzNotABug — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**